bucket/jcpicker.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "5.5",
+    "description": "Free portable offline colour picker and colour editor for web designers and digital artists.",
```

License has its own property. Portable is not needed.

```suggestion
    "description": "Color picker and color editor for web designers and digital artists",
```

or without target audience as it is useless and does not provide any added value for end user

```suggestion
    "description": "Color picker and color editor",
```